client/client.go

```diff
@@ -105,7 +106,9 @@ func (r *NotaryRepository) Initialize(rootKeyID string) error {
 		return err
 	}
 
-	rootCert, err := cryptoservice.GenerateCertificate(privKey, r.gun)
+	// Hard-coded policy: the generated certificate expires in 10 years.
+	startTime := time.Now()
+	rootCert, err := cryptoservice.GenerateCertificate(privKey, r.gun, startTime, startTime.AddDate(10, 0, 0))
```

Comment:
I wonder if it might be easier to find this policy if we used `Time.Add(Duration)`, and specified the duration as a `const` near the top of the file?